phone_number.rb

```diff
@@ -0,0 +1,63 @@
+require_relative "./phone_actions"
+
+class PhoneNumber
+  USERNAME = ENV['ACTIONS_USERNAME']
+  PASSWORD = ENV['ACTIONS_PASSWORD']
+
+  def initialize(number)
+    @number = number
+    @number = format_number
```

Comment:
since this class assumes a 10 digit phone number, go ahead and add a guard clause in the constructor here that enforces that assumption. If we decide to extend this to support numbers longer than 10 digits, we can remove the guard at that point.

Should look something like:
```suggestion
  def initialize(number)
  raise 'Must be 10 digit phone number' unless number.gsub!(/[^0-9]/, "").length == 10
```